src/mir/well-formed-mir.rkt

```diff
@@ -83,9 +119,35 @@
    (well-formed/Rvalue Γ (use Operand))
    ]
 
+  [(well-formed/Operand Γ Operand)
+   ----------------------------------------
+   (well-formed/Rvalue Γ (repeat Operand _))
+   ]
+
+  [(well-formed/Place Γ Place)
+   ----------------------------------------
+   (well-formed/Rvalue Γ (ref _ _ Place))
```

Comment:
Should I check whether the lifetime is in the set of existential vars of the body?

Reply:
We should check that the Lt is well-formed, which means either in the set of existential vars or one of the in-scope universal variables, I think.

Reply:
Done. Do we also need to check well-formedness of the types of `LocalDecls`?

Reply:
Yes, we ought to prove that, but let's file an issue for it, because i'm working a lot on our approach to well-formedness right now and some of the helper functions I think you need don't exist yet =)